contact.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+$nameErr = $priceErr = $minCountErr = $maxCountErr = "";
+$name = $price = $playerCountMin = $playerCountMax = $image = $gameType = "";
+
+if (
+    isset($_POST['name']) && isset($_POST['price']) && isset($_POST['playerCountMin'])
+    && isset($_POST['playerCountMax']) && isset($_POST['image']) && isset($_POST['gameType'])
+) {
+
+    $name = $_POST['name'];
+    $price = $_POST['price'];
+    $playerCountMin = $_POST['playerCountMin'];
+    $playerCountMax = $_POST['playerCountMax'];
+    $image = $_POST['image'];
+    $gameType = $_POST['gameType'];
+
+    if ($_SERVER["REQUEST_METHOD"] == "POST") {
+
+        if (empty($_POST["name"])) {
+            $nameErr = "Name is required";
+        } else {
+            $name = ($_POST["name"]);
+        }
+        if (empty($_POST["price"])) {
+            $priceErr = "RRP is required";
+        } else {
+            $price = ($_POST["price"]);
+        }
+        if (empty($_POST["playerCountMin"])) {
+            $minCountErr = "Minimum no. of players is required";
+        } else {
+            $playerCountMin = ($_POST["playerCountMin"]);
+        }
+        if (empty($_POST["playerCountMax"])) {
+            $maxCountErr = "Maximum no. of players is required";
+        } else {
+            $playerCountMax = ($_POST["playerCountMax"]);
+        }
+        //check no error messages - all fields are complete: 
+        //if none then submit to database and move to homepage - still need to write code to send to db!
+
+        if ($nameErr == "" && $priceErr == "" && $minCountErr == "" && $maxCountErr == "") {
+            header('Location: index.php');
+        }
+
+        $db = new PDO('mysql:host=db; dbname=collections_display', 'root', 'password');
+
+        $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+        $query = $db->prepare("INSERT INTO board_games (`name`, `rrp`, `min_player_count`, `max_player_count`, `image`, `type`) 
+            VALUES ('$name', '$price', '$playerCountMin', '$playerCountMax', '$image', '$gameType')");
+
+        $query->execute();
```

Comment:
This should also ultimtely end up in a model, but as per my other comment, I would suggest you start by trying to move the SELECT query into a model first

Reply:
Is it ok to commit the work done so far and have this be merged to main - I haven't moved the above code into a model but feel I'm running out of time today and want code so far committed to main before sprint review if possible? I will carry on working on the functionality and OOP stuff in my own time.